[2025-02-23T15:52:38.294+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:52:38.333+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-04T00:00:00+00:00 [queued]>
[2025-02-23T15:52:38.360+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-04T00:00:00+00:00 [queued]>
[2025-02-23T15:52:38.362+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T15:52:38.399+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): third_task> on 2025-02-04 00:00:00+00:00
[2025-02-23T15:52:38.417+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1430) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T15:52:38.421+0000] {standard_task_runner.py:72} INFO - Started process 1460 to run task
[2025-02-23T15:52:38.414+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'third_task', 'scheduled__2025-02-04T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmptd74tocm']
[2025-02-23T15:52:38.446+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask third_task
[2025-02-23T15:52:38.523+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.third_task scheduled__2025-02-04T00:00:00+00:00 [running]> on host 2cebbf768e94
[2025-02-23T15:52:38.663+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-04T00:00:00+00:00'
[2025-02-23T15:52:38.667+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:52:38.704+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T15:52:38.709+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo finally, it will be done!!!']
[2025-02-23T15:52:38.723+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T15:52:38.724+0000] {subprocess.py:106} INFO - finally, it will be done!!!
[2025-02-23T15:52:38.725+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T15:52:38.752+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:52:38.752+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=third_task, run_id=scheduled__2025-02-04T00:00:00+00:00, execution_date=20250204T000000, start_date=20250223T155238, end_date=20250223T155238
[2025-02-23T15:52:38.810+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:52:38.840+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:52:38.847+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-24T04:47:13.553+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:47:13.609+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-04T00:00:00+00:00 [queued]>
[2025-02-24T04:47:13.629+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-04T00:00:00+00:00 [queued]>
[2025-02-24T04:47:13.630+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T04:47:13.751+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): third_task> on 2025-02-04 00:00:00+00:00
[2025-02-24T04:47:13.768+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=833) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:47:13.774+0000] {standard_task_runner.py:72} INFO - Started process 852 to run task
[2025-02-24T04:47:13.781+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'third_task', 'scheduled__2025-02-04T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpg7280wx6']
[2025-02-24T04:47:13.786+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask third_task
[2025-02-24T04:47:13.907+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.third_task scheduled__2025-02-04T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:47:14.007+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-04T00:00:00+00:00'
[2025-02-24T04:47:14.009+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:47:14.034+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T04:47:14.037+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo finally, it will be done!!!']
[2025-02-24T04:47:14.059+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T04:47:14.061+0000] {subprocess.py:106} INFO - finally, it will be done!!!
[2025-02-24T04:47:14.062+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T04:47:14.128+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:47:14.136+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=third_task, run_id=scheduled__2025-02-04T00:00:00+00:00, execution_date=20250204T000000, start_date=20250224T044713, end_date=20250224T044714
[2025-02-24T04:47:14.326+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:47:14.399+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-24T04:47:14.411+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
